ecosystem/sep-0030.md

```diff
@@ -257,7 +258,10 @@ Name | Type | Description
 `identities` | array | A list of identities that if any one is authenticated will be able to gain control of this account.
 `identities[].role` | string | Same as request. See above.
 `identities[].authenticated` | boolean | Optional. Present and true if the currently authenticated client is authenticated with the identity.
-`signer` | string | The signer public key that the server will sign transactions with for this account when requested by an authenticated user.
+`signer` | string | The most recently added signer public key that the server will sign transactions with for this account when requested by an authenticated user.
+`signers` | array | The signers' public keys that the server can sign transactions with for this account when requested by an authenticated user, ordered from most recently added to least recently added.
+`signers[].signer` | array | The signer public key that the server can sign transactions with for this account when requested by an authenticated user.
```

Comment:
Thinking more about this, I also feel like:

- having `signer` and `signers` in the same object might be confusing
- since we also display timestamps for each signer, it might give the false impression that only the one under `signer` is active, and the other ones are basically a history of previous keys.

My suggestion to address this:

```javascript
{
    // [...]
    "signer": "GDAF..." // here as a shortcut, and also to ensure backwards compatibility
    "available_signers": [
        { "public_key": "GDAF...",  "generated_at": "..." },
        { "public_key": "G5A0...",  "generated_at": "..." }
    ]
}

Reply:
I had the same thoughts about calling it `public_key` too. I went with `signer` because I wanted the field name to be the same everywhere a signing public key is specified rather than `public_key` here, and `signer` elsewhere. The field is called `signer` in the /accounts/sign endpoints. I could rename those fields to `public_key` too though 💭. What do you think?

Reply:
I saw that you added a proposal on leighmcculloch#4 to replace with `key`, and I like that. Although it's still ambiguous whether that's the _public_ key or _private_ one, but I guess it's obvious that private keys shouldn't be exposed anywhere?

Reply:
I copied the term `key` from the Horizon API that has a `signers` list and within each signer is a `key` param. Technically not all signers are public, while in the context of this SEP I doubt they would ever not be public keys, it's possible there might be a use case with other key types (transaction hash, hash, etc).

Reply:
I've updated the PR so this PR marks some things deprecated and adds new fields. leighmcculloch#4 contains the removal of deprecated things only. Get's us to the same place but we can migrate in two steps.